cli-plugins/manager/metadata.go

```diff
@@ -0,0 +1,20 @@
+package manager
+
+var (
+	// NamePrefix is the prefix required on all plugin binary names
+	NamePrefix = "docker-"
+
+	// MetadataSubcommandName is the name of the plugin subcommand
+	// which must be supported by every plugin and returns the
+	// plugin metadata.
+	MetadataSubcommandName = "docker-cli-plugin-metadata"
+)
+
+// Metadata provided by the plugin
```

Comment:
Maybe add more description here, as it is something plugin writers will have to use/fill (even if the fields are self-explanatory)?

Reply:
Would a ref to docs/extend/cli_plugins.md be enough?

Reply:
I think it should be there for various reasons:
- godoc
- IDE auto-completion with documentation